app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/teidata/TEIDataPresenter.kt

```diff
@@ -359,17 +362,30 @@ class TEIDataPresenter(
         }
     }
 
-    private fun manageAddNewEventOptionSelected(eventCreationType: EventCreationType, stage: ProgramStage?) {
+    private fun manageAddNewEventOptionSelected(
+        eventCreationType: EventCreationType,
+        stage: ProgramStage?,
+    ) {
         if (stage != null) {
             when (eventCreationType) {
                 EventCreationType.ADDNEW -> programUid?.let { program ->
                     checkOrgUnitCount(program, stage.uid())
                 }
 
+                EventCreationType.SCHEDULE -> {
+                    view.displayScheduleEvent(
+                        programStage = stage,
+                        showYesNoOptions = false,
+                    )
+                }
+
                 else -> view.goToEventInitial(eventCreationType, stage)
             }
         } else {
-            createEventInEnrollment(eventCreationType)
+            view.displayScheduleEvent(
```

Comment:
The method that you changed `createEventInEnrollment` it's only used in a test. What do you think about removing it?.
Also this method was navigating to `ProgramStageSelectionActivity` which ( I think is no used anymore). Could you check and if so, remove it?
Thanks!.

Reply:
Done ✅